.github/workflows/lint_push.yml

```diff
@@ -0,0 +1,129 @@
+name: Lint Source
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  lint-shellcheck:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Runs shell script static analysis
+        run: |
+          sudo apt-get install shellcheck
+          ./run-tests.sh --check-shellscript
+
+  lint-pydocstyle:
```

Comment:
should we check python commands only on python/backend changes?

Reply:
We are checking this on push only for whole code.

We are using darker for only changes introduced in the PR.

Reply:
ok true. but we need to lint/check commit messages before we merge..so maybe we move "commit message" part in a global one